_data/rules.yml

```diff
@@ -391,6 +391,32 @@ groups:
                 severity: critical
                 for: 2m
 
+      - name: VMware
+        exporters:
+          - name: pryorda/vmware_exporter
+            doc_url: https://github.com/pryorda/vmware_exporter
+            rules:
+              - name: Virtual Machine Memory Warning
+                description: 'High memory usage on {{ $labels.instance }}: {{ $value | printf "%.2f"}}%'
+                query: '((vmware_vm_mem_usage_average / 100) >= 90) and ((vmware_vm_mem_usage_average / 100) < 95)'
+                severity: warning
+                for: 30m
```

Comment:
`5m` 

This alert will never trigger, because if memory goes 89% during 1 second (eg: OOM killer), then it will break the 30min check.